src/cppsim/qubit_info.hpp

```diff
@@ -54,6 +54,12 @@ class DllExport ControlQubitInfo : public QubitInfo {
      * @return コントロール値
      */
     UINT control_value() const { return _control_value; }
+    /**
+     * \~japanese-en コンストラクタ
+     *
+     * 初期化に必要、万が一使われたとき落ちるために9999にしてある
+     */
+    ControlQubitInfo(void) : QubitInfo(9999), _control_value(1){};
```

Comment:
初期化に使うなら、コメントではなくコードで初期値を示したほうが良いと思います。
10行目あたりに定数を定義しておいて、それを使うのはどうでしょうか？
```suggestion
const UINT invalid_qubit = 9999; //これを10行目あたりで定義しておく
    ControlQubitInfo(void) : QubitInfo(invalid_qubit), _control_value(1){};
```